### ISSUE TITLE
Import from wiki

### ISSUE THREAD

**Issue by kazk** [codewars/docs#5]
We have some documentation in our [wiki](https://github.com/codewars/codewars.com/wiki). Import the pages that makes sense, restructuring as necessary.

See [wiki/](https://github.com/codewars/docs/tree/master/wiki) for the files to work on.

**Comment by Blind4Basics** [comment]
How are we supposed to do this? Copy/paste the text into a PR, somehow? (Yeah, outside of cw's katas, I'm absolutely "green"... x) )

**Comment by kazk** [comment]
We can start by creating issues for specific tasks. Then work from there.
For example, I created #17 for Python test framework.

**Comment by hobovsky** [comment]
I know we have old wiki available at `/docs/wiki`, but I believe it was transferred once and when original document will be updated, we need to synchronize it again if we want to migrate it?
For example I updated "Troubleshooting" FAQ on original wiki, so I need to remember to migrate the latest version and one at `/docs/wiki` will be outdated, am I right?

**Comment by kazk** [comment]
Yeah, edited pages are recorded on Zulip topic and that's probably the only one that changed.

**Comment by hobovsky** [comment]
Is https://github.com/codewars/docs/blob/master/wiki/error-messages-submission-timed-out.md still relevant? I haven't seen such error in ages, and users also do not report it anymore. I guess the article was created when performance of CodeWars was much worse and today things are much better. Do we still need this page? Can we get rid of it if not?

**Comment by kazk** [comment]
Yeah, let's get rid of it. I don't think it has happened since the runner was rewritten.

**Comment by hobovsky** [comment]
Still to be transferred:

## About
- [ ] [Kumite](https://github.com/codewars/codewars.com/wiki/Kumite)
- [ ] [Clans, Allies & Followers](https://github.com/Codewars/codewars.com/wiki/Clans,-Allies-&-Followers)
## Reference
- [ ] [Kata Trainer](https://github.com/codewars/codewars.com/wiki/Kata-Trainer) 
- [ ] [Kata Creator](https://github.com/codewars/codewars.com/wiki/Kata-Creator)
## Tutorials
- [ ] [Creating your first kata](https://github.com/codewars/codewars.com/wiki/Tutorial%3A-Create-Your-First-Kata)
- [ ] [Troubleshooting your solution](https://github.com/codewars/codewars.com/wiki/Troubleshooting-your-solution)
## Snippets
- [ ] [Random testing in JavaScript](https://github.com/codewars/codewars.com/wiki/Snippets%3A-Random-testing-in-JavaScript)
## Kata QA



I will create issues about them and prioritize them somehow.

**Comment by hobovsky** [comment]
@kazk I have no good idea how to "migrate" the "Kata QA" part: lists of kata with some problems. Keep these as lists in docs, or maybe create dedicated Kata QA boards?

For others, I've created [new project board](https://github.com/codewars/docs/projects/1) which you (and anyone else) can use to prioritize the tasks so authors (probably me) could pick them for processing in some order.

**Comment by kazk** [comment]
I wasn't planning to migrate that because those are temporary, and it's better to keep them editable by anyone to indicate progress. Maybe the docs can have a page about helping to maintain existing kata, and include a link to a wiki page listing the lists.

I had dumped all the pages I wanted to rewrite/import under [`/__tmp__`](https://github.com/codewars/docs/tree/master/__tmp__) (originally named "wiki" as I wrote when I opened this issue).

**Comment by hobovsky** [comment]
Oh and I thought that you just removed the `wiki` folder, I havent noticed it got renamed.

Do you have any priorities w.r.t. what should be migrated/written/added in the first order? If not, I will just continue picking topics from the list as I feel like.

**Comment by kazk** [comment]
Sounds good to me. I can't think of any priorities at the moment.

**Comment by hobovsky** [comment]
Can content of `__tmp__` folder be modified, or you want to keep it unchanged as some kind of history or reference? Maybe removing migrated files or clearing migrated parts would help to keep track of what's already been processed?

**Comment by kazk** [comment]
> Maybe removing migrated files or clearing migrated parts would help to keep track of what's already been processed?

This was my intention. You can do whatever you want with them.
